tri.tf

```
@@ -0,0 +1,17 @@
+locals {
+  tri_team = [
+    "ahcorde",
+    "caguero"
```

Add a trailing comma to avoid a line change when adding additional members.
```suggestion
    "caguero",
```